charts/argo-workflows/templates/controller/workflow-rb.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.workflow.rbac.create -}}
-  {{- range $namespace := or .Values.singeNamespace false | ternary (list "") (append .Values.controller.workflowNamespaces (coalesce .Values.workflow.namespace .Release.Namespace) | uniq)  }}
+  {{- range $namespace := or .Values.singleNamespace false | ternary (list "") (append .Values.controller.workflowNamespaces (coalesce .Values.workflow.namespace .Release.Namespace) | uniq)  }}
```

Comment:
A search shows this needs to be fixed in two more places:
https://github.com/argoproj/argo-helm/blob/37277315ce669a12e9a5a66ce03ab2a6e08eb184/charts/argo-workflows/templates/controller/workflow-sa.yaml#L2
https://github.com/argoproj/argo-helm/blob/37277315ce669a12e9a5a66ce03ab2a6e08eb184/charts/argo-workflows/templates/controller/workflow-role.yaml#L2

Reply:
:laughing: classic copy-paste error

Reply:
Resolved in 5830d82263ace450a909d461f93e1ba806645601